### PR TITLE
[enocean] Support EEP A5-13 for environmental sensors

### LINKED ISSUE #5740: [enocean] Support for environmental sensors (EEP A5-13)
The EnOcean binding should also support environmental sensors like the Eltako Multisensor MS (incl. FWS61) with EEP A5-13.

### PR DESCRIPTION
**Pull request by fruggy83**
Implemented EEP A5-13 to support environmental sensors like Eltako Multisensor MS.

Fixes #5740

### CODE REVIEW

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.eep.A5_13;
+
+import org.openhab.binding.enocean.internal.eep.Base._4BSMessage;
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+
+/**
+ *
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java]: Every class should have description about its goal / intention

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    protected boolean isPartOne() {
+        return getMessageIdentifier() == PARTONE;
+    }
+
+    protected boolean isPartTwo() {
+        return getMessageIdentifier() == PARTTWO;
+    }
+}
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java]: Please make sure all your files contain a newline at the end of the file

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.library.unit.SmartHomeUnits;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+
+/**
+ *
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java]: See above

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                case CHANNEL_ILLUMINATIONSOUTHNORTH:
+                    return getIlluminationSouthNorth();
+                case CHANNEL_ILLUMINATIONEAST:
+                    return getIlluminationEast();
+            }
+        }
+
+        return UnDefType.UNDEF;
+    }
+}
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java]: see above

File: bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+				<label>EEP</label>
+				<description>EEP which is used by sensor</description>
+				<options>
+					<option value="A5_13_01">A5-13-01 Weather station</option>
+				</options>
+				<limitToOptions>true</limitToOptions>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml]: Please remove newlines

File: bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+				</options>
+				<limitToOptions>true</limitToOptions>
+				<required>true</required>
+			</parameter>
+		</config-description>
+		
+
+	</thing-type>
+
+</thing:thing-descriptions>
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml]: End of file newline

**Comment by cweitkamp** [comment]
@fruggy83 Did you see the comments?

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java (hunk 1 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java]: should be solved

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java (hunk 2 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java]: should be solved

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java (hunk 3 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java]: should be solved

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java (hunk 4 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13_01.java]: should be solved

File: bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml (hunk 5 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml]: should be solved

File: bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml (hunk 6 above)
Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/EnvironmentalSensor.xml]: should be solved

**Comment by fruggy83** [comment]
Thanks for your review @martinvw and your hint @cweitkamp. I am really sorry for this delay, I just lost sight of this PR.

**Comment by kaikreuzer** [comment]
> @martinvw I leave the merge to you so that you can check that your comments are all addressed.

@martinvw Friendly ping!

**Comment by fruggy83** [comment]
@martinvw kindly pinging for a second review. Thanks a lot in advance

**Comment by martinvw** [comment]
@fruggy83 sorry :-) Will try to get back to you asap.

**Comment by martinvw** [comment]
@fruggy83 could you please resolve the conflict? Thanks!

**Comment by fruggy83** [comment]
@martinvw I did a rebase on the current master, this should solve the conflicts. Thanks a lot for your review 👍

**Comment by martinvw** [comment]
Thanks @fruggy83

**Pull request merged**